vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/LazyLoadingIT.java

```diff
@@ -96,85 +96,6 @@ public void selectItem_changeFilter_properlyFilteredItems() {
         assertNotRendered("Item 2");
     }
 
-    @Test
-    public void scrollOverlay_morePagesLoaded_overflowingPagesDiscarded() {
-        stringBox.openPopup();
-        scrollToItem(stringBox, 50);
-        waitUntilTextInContent("Item 52");
-
-        assertLoadedItemsCount(
-                "There should be 100 items after loading two pages", 100,
-                stringBox);
-        assertRendered("Item 52");
-
-        scrollToItem(stringBox, 100);
-        waitUntilTextInContent("Item 102");
-
-        assertLoadedItemsCount(
-                "There should be 150 items after loading three pages", 150,
-                stringBox);
-        assertRendered("Item 102");
-
-        // The first pages should get discarded (active range has default limit
-        // of 500)
-        for (int i = 150; i <= 600; i += 50) {
-            scrollToItem(stringBox, i);
-            waitUntilTextInContent("Item " + i);
-        }
-
-        assertLoadedItemsCount(
-                "There should be 500 items after loading multiple pages", 500,
-                stringBox);
-        assertRendered("Item 602");
-
-        // The last pages should get discarded (active range has default limit
-        // of 500)
-        for (int i = 600; i >= 0; i -= 50) {
-            scrollToItem(stringBox, i);
-            waitUntilTextInContent("Item " + i);
-        }
-
-        assertLoadedItemsCount(
-                "There should be 500 items after scrolling back to start", 500,
-                stringBox);
-        assertRendered("Item 2");
-    }
-
-    @Test
-    public void openPopup_scrollToEnd_onlyLastPageLoaded() {
-        stringBox.openPopup();
-        scrollToItem(stringBox, 1000);
-        waitUntil(e -> getOverlayContents().contains("Item 999"));
-        assertLoadedItemsCount(
-                "Expected the last page to be loaded (50 items).", 50,
-                stringBox);
-        assertRendered("Item 999");
-    }
-
-    @Test
-    public void scrollToEnd_scrollUpwards_pagesLoaded() {
```

Comment:
This test is covered by `defaultPageSize_scrollToEnd_scrollUpAndDown_morePagesLoaded_overflowingPagesDiscarded`